tldw_Server_API/app/core/DB_Management/media_db/runtime/rows.py

```
@@ -0,0 +1,90 @@
+        return self._data.get(key, default)
+
+    def keys(self):
+        return self._data.keys()
+
+    def values(self):
+        return self._data.values()
+
+    def items(self):
+        return self._data.items()
+
+    def __iter__(self):
```

Ruff (0.15.6)

[warning] Missing return type annotation for special method `__iter__`

(ANN204)